modules/api/src/main/java/org/apache/ignite/tx/TransactionOptions.java

```diff
@@ -39,10 +39,14 @@ public long timeoutMillis() {
     /**
      * Sets transaction timeout, in milliseconds.
      *
-     * @param timeoutMillis Transaction timeout, in milliseconds.
+     * @param timeoutMillis Transaction timeout, in milliseconds. Cannot be negative; 0 means 'use default timeout'.
```

Comment:
Let's explain where the default timeout comes from.

Reply:
Added an explanation both here and in the C# counterpart